SamplifyAPIClientTest/Tester.cs

```diff
@@ -4,110 +4,137 @@
 using Newtonsoft.Json;
 using Dynata.SamplifyAPIClient;
 using Xunit;
+using System.IO;
+
 namespace SamplifyAPIClientTest
 {
     public class TempTest
     {
         [Fact]
-        public async Task TestAll()
```

Comment:
We probably don't want to keep this file as part of the test project. If we do, can we fix this file.

Reply:
Fixed.